pkg/util/git.go

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
We may unify this file with the one from `pkg/notes/git.go`

Reply:
Agreed here, `git` related things feel like they would be worthy of their own package

Reply:
+1! That was exactly what I had in mind. :)